server/src/main/clients/web-browser/src/pages/Spinner.tsx

```diff
@@ -0,0 +1,74 @@
+import {useEffect, useState} from 'react';
+import * as protos from '../protos';
+import {Coordinate, IkigaiReel} from '../IkigaiReel/IkigaiReel';
+
+export default function Spinner() {
```

Comment:
This was intended to be a temporary page because I needed to replace the front page but I didn't want to delete your work. Should I put it somewhere else, like commented out in `Root.tsx`? Or I could just delete it and find it in the git history later?

Reply:
Temporarily moved to a comment in `Root.tsx`, will add back later